packages/node/src/indexer/api.service.ts

```diff
@@ -86,13 +89,20 @@ export class ApiService
 
     this.registry = await this.buildRegistry();
 
+    const { argv } = yargsOptions;
```

Comment:
Use NodeConfig instead. It is already extensible for this, see Eth SDK if there is no example with cosmos